blockservice/blockservice.go

```diff
@@ -145,6 +145,6 @@ func (s *BlockService) DeleteBlock(k key.Key) error {
 }
 
 func (s *BlockService) Close() error {
-	log.Debug("blockservice is shutting down...")
+	defer log.Debug("blockservice is shutting down...")
```

Comment:
i havent looked at it in the console, but i dont think this should be a defer, it's a log statement so we know when we _enter_ the shutting down process...

im ok with this though if we want to switch it back we can do so later.